mavros/launch/px4_config.yaml

```diff
@@ -44,87 +47,98 @@ cmd:
 # None
 
 # global_position
-global_position:
-  frame_id: "map"             # origin frame
-  child_frame_id: "base_link" # body-fixed frame
-  rot_covariance: 99999.0   # covariance for attitude?
-  gps_uere: 1.0             # User Equivalent Range Error (UERE) of GPS sensor (m)
-  use_relative_alt: true    # use relative altitude for local coordinates
-  tf:
-    send: false               # send TF?
-    frame_id: "map"  # TF frame_id
-    global_frame_id: "earth"  # TF earth frame_id
-    child_frame_id: "base_link" # TF child_frame_id
+/mavros/global_position:
+  ros__parameters:
+    frame_id: "map"             # origin frame
+    child_frame_id: "base_link" # body-fixed frame
+    rot_covariance: 99999.0   # covariance for attitude?
+    gps_uere: 1.0             # User Equivalent Range Error (UERE) of GPS sensor (m)
+    use_relative_alt: true    # use relative altitude for local coordinates
+    tf:
+      send: false               # send TF?
+      frame_id: "map"  # TF frame_id
+      global_frame_id: "earth"  # TF earth frame_id
+      child_frame_id: "base_link" # TF child_frame_id
 
 # imu_pub
-imu:
-  frame_id: "base_link"
-  # need find actual values
-  linear_acceleration_stdev: 0.0003
-  angular_velocity_stdev: 0.0003490659 // 0.02 degrees
-  orientation_stdev: 1.0
-  magnetic_stdev: 0.0
+/mavros/imu:
+  ros__parameters:
+    frame_id: base_link
+    # need find actual values
+    linear_acceleration_stdev: 0.0003
+    angular_velocity_stdev: 0.0003490659 # 0.02 degrees
+    orientation_stdev: 1.0
+    magnetic_stdev: 0.0
 
 # local_position
-local_position:
-  frame_id: "map"
-  tf:
-    send: false
+/mavros/local_position:
+  ros__parameters:
     frame_id: "map"
-    child_frame_id: "base_link"
-    send_fcu: false
+    tf:
+      send: false
+      frame_id: "map"
+      child_frame_id: "base_link"
+      send_fcu: false
 
 # param
 # None, used for FCU params
 
 # rc_io
 # None
 
-# safety_area
-safety_area:
-  p1: {x:  1.0, y:  1.0, z:  1.0}
-  p2: {x: -1.0, y: -1.0, z: -1.0}
+# # safety_area
+/mavros/safety_area:
```

Comment:
Safety area is quite deprecated feature. I've removed it from ROS2.

Reply:
OK, I see.